### ISSUE TITLE
Backup materialized view with postgresql

### ISSUE THREAD

**Issue by michaelsogos** [dbeaver/dbeaver#4148]
Dear All,

Seems that when we starts creating backup, is possible to select TABLES and VIEWS to backup but not MATERIALIZED VIEW on postgresql.

**Comment by JamesInform** [comment]
Hi Michael,

DBeaver is a decent tool. But if it comes to backup/restore PostgreSQL databases or parts of it, it is always good to use the existing PostgreSQL tools like pg_dump, pg_dump_all and pg_restore.

Please keep in mind that Materialized Views are filled from the underlaying tables. So the backup strategy is to save the underlaying table data and refresh the materialized view on restore.

Greets
James

**Comment by michaelsogos** [comment]
Hi @JamesInform,

we are using these tools, but also I guess that dbeaver can improve thanks of community suggestions because in my opinion it is more than a decent tool.

in our company we tried many tools but just dbeaver gave as great impression and we started to adopt it as our default db tool.

Of course it is not part of our backup strategy, however sometimes we have to make backups and we guess it can be improved.
